### PR TITLE
feat(editor): SSO onboarding

### PR DESCRIPTION
**Pull request by cstuncsik**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

**Comment by krynble** [comment]
Is this PR valid or should we close it @cstuncsik ?

**Comment by cstuncsik** [comment]
It's valid, going to finish it today

**Review by flipswitchingmonkey — APPROVED** [review]

File: packages/editor-ui/src/stores/sso.ts [hunk 1]
```diff
@@ -58,6 +61,13 @@ export const useSSOStore = defineStore('sso', () => {
 		ssoApi.saveSamlConfig(rootStore.getRestApiContext, config);
 	const testSamlConfig = () => ssoApi.testSamlConfig(rootStore.getRestApiContext);
 
+	const updateUser = async (params: { firstName: string; lastName: string }) =>
+		updateCurrentUser(rootStore.getRestApiContext, {
+			id: usersStore.currentUser!.id,
```
[the remark this answers is not in the document]
Comment (cstuncsik) [reply packages/editor-ui/src/stores/sso.ts:66]: As I have seen it in the `me` controller it throws only if the current and the request email are different (also if there is no email at all)
Maybe we should introduce a new REST path just for the onboarding

**Comment by janober** [comment]
Got released with [`n8n@0.223.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.223.0)

**Pull request merged**